libsplinter/src/admin/rest_api/actix/ws_register_type.rs

```diff
@@ -26,7 +26,7 @@ use crate::admin::service::{
     AdminCommands, AdminServiceEventSubscriber, AdminServiceStatus, AdminSubscriberError,
 };
 #[cfg(feature = "admin-service-event-store")]
-use crate::admin::store::events;
```

Comment:
Typo in commit message `This includes a removing `

Reply:
Ended up re-wording this commit after updating the admin service to still take an "event_store" (that uses the admin store's trait) so I believe this sentence/typo was removed entirely.